contrib/awssqs/samples/dumper-resources.yaml

```diff
@@ -23,8 +36,7 @@ spec:
 apiVersion: serving.knative.dev/v1alpha1
 kind: Service
 metadata:
-  name: message-dumper
-  namespace: default
+  name: awssqs-message-dumper
```

Comment:
let's use the new display events thing from @n3wscott 

https://github.com/knative/eventing-sources/pull/249

Reply:
I was watching that one, my plan was to update once it was merged.

Reply:
done